### ISSUE TITLE
how can i use vue/no-deprecated-props-default-this in vue2 project

### ISSUE THREAD

**Issue by hellozhangran** [vuejs/eslint-plugin-vue#1835]
**Please describe what the rule should do:**

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue

```

**Additional context**

**Comment by FloEdelmann** [comment]
You can enable individual rules (like this one) that will override any shared config.

See e.g. https://eslint.vuejs.org/user-guide/#usage
